quinn-proto/src/transport_parameters.rs

```diff
@@ -467,6 +475,82 @@ impl TransportParameters {
     }
 }
 
+/// A reserved transport parameter.
+///
+/// It has an identifier of the form 31 * N + 27 for the integer value of N.
+/// Such identifiers are reserved to exercise the requirement that unknown transport parameters be ignored.
+/// The reserved transport parameter has no semantics and can carry arbitrary values.
+/// It may be included in `quic_transport_parameters` TLS extensions when sent and should be ignored when received.
```

Comment:
```suggestion
/// It may be included in transport parameters sent to the peer, and should be ignored when received.
```
Transport parameters are not TLS extensions, and Quinn defines no identifier by the name `quic_transport_parameters`.

Reply:
I've made mistake with plural form of word extensions, I wanted to say that this reserved transport parameter may be included and carried inside [QUIC transport parameters extension](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#tls-extensiontype-values-1) of TLS https://www.rfc-editor.org/rfc/rfc9001#section-8.2 (at least according to my current understanding of the subject). So should I fix plural form or just accept your suggestion or maybe rephrase the sentence in a more clear way?

Reply:
I think @Ralith's suggestion is simpler and more clear.